_assets/stylesheets/vendor/_normalize.scss

```diff
@@ -1,14 +1,15 @@
-/*! normalize.css v3.0.2 | MIT License | github.com/necolas/normalize.css */
+/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
 
 /**
  * 1. Set default font family to sans-serif.
- * 2. Prevent iOS text size adjust after orientation change, without disabling
- *    user zoom.
+ * 2. Prevent iOS and IE text size adjust after device orientation change,
+ *    without disabling user zoom.
  */
 
 html {
   font-family: sans-serif; /* 1 */
-  -ms-text-size-adjust: 100%; /* 2 */
+
+      -ms-text-size-adjust: 100%; /* 2 */
```

Comment:
Use `//` comments everywhere<br>Line should be indented 2 spaces, but was indented 6 spaces